apps/nspanel-lovelace-ui/luibackend/pages.py

```diff
@@ -213,20 +213,31 @@ def generate_entities_item(self, item, cardType="cardGrid", temp_unit=""):
         uuid = item.uuid
         # type of the item is the string before the "." in the entityId
         entityType = entityId.split(".")[0]
-        
+
         apis.ha_api.log(f"Generating item for {entityId} with type {entityType}", level="DEBUG")
+
+        status_entity = (item.status and apis.ha_api.entity_exists(item.status) and apis.ha_api.get_entity(item.status)) or None

```

Comment:
This is how the and and or operators are defined: https://docs.python.org/3/reference/expressions.html#boolean-operations
First the left operand is evaluated and then, depending on its truty-ness, either returned or the right operand is evaluated and then returned.

But I see how this may be confusing and am preparing an alternative using `if`/`then`/`else`.

Reply:
this is acually nice, however really unexpeced if you don't know